CONTRIBUTING.md

```diff
@@ -410,66 +410,79 @@ If you are adding a new service directory, ensure that it is mapped to a friendl
 
 ## On-boarding New generated code library
 
-1. Make a copy of `/sdk/template/Azure.Template` in you appropriate service directory and rename projects to `Azure.Management.*` for management libraries or `Azure.*` (e.g.  `sdk/storage/Azure.Management.Storage` or `sdk/storage/Azure.Storage.Blobs`)
+1. Make a copy of `sdk/template/Azure.Template` in you appropriate service directory and rename projects to `Azure.Management.*` for management libraries or `Azure.*` (e.g.  `sdk/storage/Azure.Management.Storage` or `sdk/storage/Azure.Storage.Blobs`).
+
+   Alternatively, you can install `sdk/template` as a `dotnet new` template to create or update a service directory:
+
+   ```bash
+   dotnet new --install /sdk/template
+   dotnet new azuresdk --name Azure.MyService --output sdk/myservice --ServiceDirectory myservice --ProjectName Azure.MyService
```

Comment:
Does name and ProjectName need to be different?

Reply:
Can we default one to another?

Reply:
I think it's too presumptuous to assume the second token is the service directory. Look at Key Vault. "KeyVault" is the third token.

Ideally, these should be global variables passed to the template engine and I wouldn't need them, since I could get them from the `--output` and `--name` standard options. Unfortunately, that's not possible right now, though there is a bug to track the work: dotnet/templating#1818

Reply:
I was talking about these two: --name Azure.MyService --ProjectName Azure.MyService which are exactly the same but it seems they too are affected by the issue you linked.